extensions/webjars-locator/runtime/src/main/java/io/quarkus/webjar/locator/runtime/WebJarLocatorHandler.java

```diff
@@ -0,0 +1,55 @@
+package io.quarkus.webjar.locator.runtime;
+
+import java.util.Map;
+
+import io.quarkus.runtime.annotations.Recorder;
+import io.vertx.core.Handler;
+import io.vertx.ext.web.RoutingContext;
+
+@Recorder
+public class WebJarLocatorHandler {
+
+    private String webjarsRootUrl;
```

Comment:
These should be parameters on the getHandler() method, recorders are not really supposed to be stateful.